tests/e2e-pw/specs/shopper/shopper-myaccount-saved-cards.spec.ts

```diff
@@ -21,11 +21,43 @@ import {
 } from '../../utils/shopper';
 import RestAPI from '../../utils/rest-api';
 
+type TestVariablesType = {
+	[ key: string ]: {
+		card: typeof config.cards.basic;
+		address: {
+			country: string;
+			postalCode: string;
+		};
+		products: [ string, number ][];
+	};
+};
+
```

Comment:
If you don't want to define the whole type, it's enough to cast products like `products: [ [ 'Cap', 1 ] ] as [ string, number ][]`

Reply:
Yep, true, see my answer on the other comment :)